tests/unit/src/kakarot/instructions/test_environmental_information.py

```diff
@@ -103,3 +103,17 @@ async def test_excodecopy_should_handle_address_with_code(
         expected = (bytecode + [0] * (offset + size))[offset : (offset + size)]
 
         assert memory_result == expected
+
+    @pytest.mark.parametrize(
+        "zeroes,nonzeroes",
+        [(4, 0), (0, 4), (4, 4)],
+        ids=["only_zeroes", "only_nonzeroes", "both_zeroes_and_nonzeroes"],
+    )
+    async def test_gasprice(self, environmental_information, zeroes, nonzeroes):
+        random_nonzeroes = [random.randint(1, 255) for _ in range(nonzeroes)]
+        calldata = [0] * zeroes + random_nonzeroes
+        random.shuffle(calldata)
```

Comment:
Make sure to set the seed when using random in test to have a deterministic behavior